rollup/ccc/async_checker.go

```diff
@@ -94,8 +99,10 @@ func (c *AsyncChecker) Wait() {
 // Check spawns an async CCC verification task.
 func (c *AsyncChecker) Check(block *types.Block) error {
 	if block.NumberU64() > c.currentHead.Number.Uint64()+1 {
-		log.Error("non continuous chain observed in AsyncChecker", "prev", c.currentHead, "got", block.Header())
-	} else if block.ParentHash() != c.currentHead.Hash() {
+		log.Warn("non continuous chain observed in AsyncChecker", "prev", c.currentHead, "got", block.Header())
```

Comment:
When can this happen in practice?

Reply:
In sequencer it should never happen that is why I added the log to verify my assumption. In follower nodes it can happen during reorgs.

Reply:
Oh, I was thinking this will happen during "graceful restart" (in `checkHeadRowConsumption`), is that not the case?

Reply:
Yeah that would be a pain to deal with. Added error handling https://github.com/scroll-tech/go-ethereum/pull/982/commits/843f5fa12276c314b73ce5d424a57aae0641a56f

Reply:
If I am not mistaken, it shouldn't happen in `checkHeadRowConsumption`. Because if `checkHeadRowConsumption` ends up spawning checker jobs, it will still do it for a continous chain of blocks. It should never skip a block in-between and spawn a task for the next one. (ie spawn for N, N+2 but skip N+1)